process-engine/bpmn-studio
# ⚡️Use getAllSuspendedTasks

## Changes

1. Use getAllSuspendedTasks

## Issues

Related to #1594
Related to https://github.com/process-engine/process_engine_runtime/issues/380

PR: #1744

## How to test the changes

- Go to the inspect view
- **Notice that the task list still get displayed correctly.**

> Do the same with the current ProcessEngine version
